lectures/linear_equations.md

```diff
@@ -16,24 +16,20 @@ kernelspec:
 ```{index} single: Linear Equations and Matrix Algebra
 ```
 
-```{contents} Contents
-:depth: 2
-```
-
 ## Overview
 
 Many problems in economics and finance require solving linear equations.
 
 In this lecture we discuss linear equations and their applications.
 
-To illustrate the importance of linear equations, we begin with a two good
+To illustrate the importance of linear equations, we begin with a two goods
```

Comment:
actually "good" is correct --- I can't explain the grammar because I'm not sure why myself.